functions/private/Test-WinUtilPackageManager.ps1

```diff
@@ -20,13 +20,16 @@ function Test-WinUtilPackageManager {
     $status = "not-installed"
 
     if ($winget) {
-        # Install Winget if not detected
-        $wingetExists = Get-Command -Name winget -ErrorAction SilentlyContinue
+        # Get Winget Version and Install it if not detected
+        $wingetVersionFull = ""
```

Comment:
```suggestion
```
You don't really need to initialize the variable as an empty string. 
Powershell will happily create the variable on first use in the try statement if it does not exist already and if you run into the catch statement, the variable won't be used either because the whole version check is skipped if winget is not installed. 
I would simply remove the line to keep the code as clean as possible :)

Reply:
> You don't really need to initialize the variable as an empty string.

After a bit of testing, it turns out PowerShell doesn't limit a Variable Initialization into a Try-Catch Scope like I thought it would, or in other words, I thought the value of `wingetVersionFull` after the Try-Catch Code Block(s), would be like accessing an uninitialized variable, and it'll null or something (Was thinking way too low level, not gonna lie 😆)

and it indeed works without that Variable Initialization on line 24, thanks for the catching my silly mistake @Marterich 😄, will commit your suggestion in a new commit 👍

### Screenshots

![image](https://github.com/ChrisTitusTech/winutil/assets/70659536/6cc2dca0-3b30-4373-a723-1271908646d2)